resume.page

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
```

Comment:
This file needs to be renamed to have a file extension ending in `.html` not `.page`.  🔨